tests/test_chordable_django_backend.py

```diff
@@ -0,0 +1,198 @@
+"""
+Test ChordableDjangoBackend.
+
+A lot of the intended usage of this backend happens asychronously, which is
+notoriously hard to test. Please do extensive testing for your situation before
+relying on async behavior, as these tests only ensure correctness in a single
+threaded process.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from builtins import range as _range
+from datetime import datetime, timedelta
+
+from celery.exceptions import ChordError
+from celery.states import FAILURE, PENDING, SUCCESS
+from djcelery.models import TaskMeta
+from freezegun import freeze_time
+from mock import MagicMock, patch
+import pytest
+
+from django.test import override_settings
+
+from celery_utils.chordable_django_backend import ChordableDjangoBackend, chord, chord_task
+from celery_utils.models import ChordData
+
+
+@chord_task()
+def chord_subtask(i):
+    """
+    Toy example of a subtask.
+    """
+    return i + i
+
+
+@chord_task()
+def chord_callback(results):
+    """
+    Toy example of a callback.
+    """
+    return sum([result for result in results])
+
+
+@chord_task()
+def chord_callback_itr(results_itr):
+    """
+    Toy example of a callback that accepts an iterator.
+    """
+    return sum([result for result in results_itr()])
```

Comment:
As above, the square brackets should be unneccesary here, and just introduce the overhead of creating a list (of potentially large size).  Not crucially important in tests, but it's good to provide quality exemplar code.